README.md

```diff
@@ -1,4 +1,29 @@
-SBI
+ScrumBoard-it
 ===
 
-A Symfony project created on April 5, 2016, 12:44 pm.
+Symfony project.
```

Comment:
Prefer warn contributor that sbi is a symfony3 application explicitly and assume from there, they know how to install an sf app.